main.lua

```diff
@@ -381,6 +383,10 @@ local function touchcheck(i, x, y)
 	end
 end
 
+local function isCtrlOrGuiDown()
+	return (love.keyboard.isDown('lctrl') or love.keyboard.isDown('lgui') or love.keyboard.isDown('rctrl') or love.keyboard.isDown('rgui'))
+end
+
```

Comment:
Is there a particular reason this function was moved up here?

Reply:
This is moved here because it’s used for muting and other stuff I haven’t committed to your rep yet. If I kept it where it was it would not work